### PR TITLE
Introduce additional lookup if the owner is kept as sigil #1721

### PR DESCRIPTION
**Pull request by TobiasNx**
Related to #1721
I introduced an additional lookup for the holding institution based on the Owner <-> Sigil <-> ISIL to catch the sublibrary ISILs, if the libraries used ther Owner as sublibrary code (or this is provided via the Bridges) One thing I recognized is that we do not have labels for certain libraries since they have no Isils anymore:
@acka47 How should I proceed here?

### CODE REVIEW

**Comment by TobiasNx** [comment]
e.g.:
{
      "id" : "http://lobid.org/organisations/DE-464#!",
      "isil" : "DE-465",
      "isil" : "DE-464",
      "label" : "lobid Organisation"
    },
    }

**Comment by acka47** [comment]
Hmm, looking at the example and resolving the ZDB URI (`http://ld.zdb-services.de/resource/organisations/DE-464`) in the browser, I am redirected to the [entry for DE-465](https://ld.zdb-services.de/resource/organisations/DE-465). Perhaps the ISIL refistry can provide us with this redirect map from deprecated ISILs to active ISILS? We could then swap the deprecated ISILs by active ones and obtain the labels for them. /cc @jorol @cklee

**Comment by cKlee** [comment]
> Perhaps the ISIL registry can provide us with this redirect map from deprecated ISILs to active ISILS?

What should this map look like? Just two columns of ISIL?

**Comment by cKlee** [comment]
Okay. We can do this once. But what to do on future redirects? We suggest that you extract this from Pica+ 035D $b when importing our data.

**Comment by TobiasNx** [comment]
@blackwinter This PR would make your #1770 outdated. I introduced a mapping for all libraries that still use the old owner as sublibrary code. This enables the right mapping for Kn-3.

But there is a general issue with regard the switch from bridge to productive alma usage which we should discuss.

**Comment by TobiasNx** [comment]
@dr0i can you update the tests please! thanks

**Review by acka47 — APPROVED** [review]
+1

File: src/test/resources/alma-fix/990126276700206441.json
Comment (blackwinter) [inline src/test/resources/alma-fix/990126276700206441.json:1]: `38/467` (`K1182`) seems to be missing in `sigel2isilMap.tsv`, should map to `DE-38-467`.

```xml
<datafield tag="ITM" ind1=" " ind2=" ">
  <subfield code="H">22105935470007147</subfield>
  <subfield code="x">00467001</subfield>
  <subfield code="f">BOOK</subfield>
  <subfield code="v">00467001</subfield>
  <subfield code="p">04</subfield>
  <subfield code="X">System</subfield>
  <subfield code="Y">2021-06-29 02:00:00 Europe/Berlin</subfield>
  <subfield code="n">467/MT/Fe40</subfield>
  <subfield code="M">49HBZ_BRIDGE_UBK</subfield>
  <subfield code="s">1</subfield>
  <subfield code="d">0</subfield>
  <subfield code="V">System</subfield>
  <subfield code="b">467/0000383</subfield>
  <subfield code="a">23105935350007147</subfield>
  <subfield code="c">467/MT/Fe40</subfield>
  <subfield code="W">2023-02-16 03:07:16 Europe/Berlin</subfield>
  <subfield code="u">K1182</subfield>
  <subfield code="w">K1182</subfield>
</datafield>
```

**Review by blackwinter — APPROVED** [review]
Looks good, except that some ISILs that are present in #1770 seem to be missing:

```
DE-38-009
DE-38-101
DE-38-111
DE-38-227
DE-38-232
DE-38-237
DE-38-326
DE-38-328
DE-38-440
DE-38-465
DE-38-466
DE-38-467
DE-38-468
```

**Comment by dr0i** [comment]
> @dr0i can you update the tests please! thanks

Just reran the build as there an download problem:
> Connecting to downloads.typesafe.com (downloads.typesafe.com)|18.165.98.46|:80... connected.
HTTP request sent, awaiting response... 403 Forbidden

**Review by dr0i — APPROVED** [review]

**Comment by dr0i** [comment]
Will be deployed next Monday.

File: src/test/resources/alma-fix/990050000600206441.json [hunk 1]
```diff
@@ -194,11 +194,11 @@
     "serialNumber" : "400887-10",
     "currentLocation" : "R3048 / FHMAT",
     "heldBy" : {
-      "id" : "http://lobid.org/organisations/DE-386#!",
-      "isil" : "DE-386",
-      "label" : "Universitätsbibliothek der RPTU in Kaiserslautern"
+      "id" : "http://lobid.org/organisations/DE-386-48#!",
+      "isil" : "DE-386-48",
+      "label" : "lobid Organisation"
```
Comment (TobiasNx) [inline src/test/resources/alma-fix/990050000600206441.json:199]: @blackwinter I was confusing Köln with this.

**Comment by blackwinter** [comment]
Looks good w.r.t. `DE-Kn3`, thanks.

**Pull request merged**